### DISCUSSION TITLE
Please provide alternative attributes in valkey-go from go-redis & confirm if selected are valid

### DISCUSSION THREAD

**Discussion by royatanu94** [valkey-io/valkey#802] [Q&A]
rcOptions = &redis.UniversalOptions{
			Addrs:             /// ClientOption.InitAddress 
			SentinelUsername:  /// ClientOption.SentinelOption.Username
			SentinelPassword: ,/// ClientOption.SentinelOption.Password
			MasterName:       ///ClientOption.SentinelOption.MasterSet
			Username:          /// ClientOption.Username
			Password:         /// ClientOption.Password
			DB:               /// ClientOption.SelectDB
			TLSConfig:        ///ClientOption.TLSConfig OR ClientOption.SentinelOption.TLSConfig
			DialTimeout:      /// ClientOption.Dialer.Timeout
			WriteTimeout:     /// ClientOption.ConnWriteTimeout
			PoolSize:         /// ClientOption.BlockingPoolSize

		PoolTimeout:   ????
	        ReadTimeout:   ????        
		}

Added redis-go replacements in comment. Please confirm if this is fine & provide missing alternatives.

**Comment by royatanu94** [comment]
@rueian Any help is highly appreciated

**Reply by rueian** [reply]
Hi @royatanu94,

There is no `PoolTimeout` equivalent and alternative in valkey-go. 

On the other hand, the alternative of `ReadTimeout` is the timeout/deadline in each `ctx` passed to each operation: https://github.com/valkey-io/valkey-go?tab=readme-ov-file#context-cancellation
